src/test/java/com/google/cloud/spanner/pgadapter/ITParameterizedQueryTest.java

```diff
@@ -1,320 +0,0 @@
-// Copyright 2021 Google LLC
```

Comment:
Removed, as this class did not implement the extended query protocol correctly (it required the backend to return a response before a sync/flush had been sent by the frontend).